app/src/main/java/fr/free/nrw/commons/wikidata/WikidataEditService.kt

```diff
@@ -55,6 +54,7 @@ class WikidataEditService @Inject constructor(
         fileEntityId: String,
         depictedItems: List<String>
     ): Observable<Boolean> {
+
```

Comment:
Please perform such stylistic changes in a separate style-only pull request. Thanks!